documentation/IDTA-01001/modules/ROOT/pages/includes/IDTA-01001_ValueOnly.adoc

```diff
@@ -418,6 +613,27 @@ For a BasicEventElement named “MyBasicEvent”, the payload is minimized to th
 }
 ----
 
+If the _BasicEventElement_ itself is the subject for ValueOnly the serialization is minimized to the following:
+
+[source,json,linenums]
+----
+{
+    "observed": {
```

Comment:
Is *observed* the only property that is considered **value** of a BasicEventElement? If so, why do we need that extra JSON object to wrap around and not simplify the serialization to
```JSON
{
     "type": "ModelReference",
      "keys": [
        {
          "type": "Submodel",
          "value": "http://customer.com/demo/aas/1/1/1234859590"
        },
        {
          "type": "Property",
          "value": "CurrentValue"
        }
      ]
    }
```

Reply:
If we would extend it by other (optional) value-attributes then the serialization would not be valid any longer

Reply:
It would not be backward-compatible to change this. So it needs to stay like this. If you want to change it please raise an issue for R4.0.